modules/devguide/examples/c/bincoding/README.md

```diff
@@ -1,7 +1,8 @@
 This example demonstrates how to write custom transcoders for structs in C.
 
-Here we store user profile with embedded avatar as document in Couchbase, and retrieve it back to render as HTML.
-Note, that encoding of the `struct Profile` remains friendly to Couchbase indexers, and data could be retrieven using any of queries (N1QL, FTS etc).
+Here we store user profile with embedded avatar as document in Couchbase, and retrieve it back to render as HTML. Note,
+that encoding of the `struct Profile` remains friendly to Couchbase indexers, and data could be retrieven using any of
+queries (N1QL, FTS etc).
 
 To try the example, use Makefile to build the binary and then execute it:
```

Comment:
reference to "Makefile" now needs updating

Reply:
So, the subdirs still need more work. These still will not build. So I didn't touch them at all. Maybe we break that out into another ticket to track the additional cleanup required? I just wasn't sure if we might already have some  (I might have already resolved some other issues in this pass).

Reply:
ah, gotcha. I added a commit in https://github.com/couchbase/docs-sdk-c/pull/112 but sounds like I jumped the gun, so I'll rebase it out now...